usr/share/rear/conf/default.conf

```diff
@@ -2169,6 +2169,28 @@ COPY_AS_IS_NBKDC=()
 COPY_AS_IS_EXCLUDE_NBKDC=()
 ####
 
+####
+# BACKUP=NFS4SERVER
+##
+# Note: This is for restore via NFS server, where another computer can connect to the rescue system via NFS and push
+#       the backup files onto the disks after ReaR prepared and formatted them.
+#
+#       This method has no backup functionality, only restore. And it requires interaction with an external system to push the files.
+#
+#       ReaR will wait for a signal file to appear to indicate that the file transfer is complete.
+#
+REQUIRED_PROGS_NFS4SERVER=("rpc.nfsd" "rpc.mountd" "exportfs" "ss" "nproc")
+NFS4SERVER_MODULES=(nfs nfsd)
+# List of share options for the recovery system, see 'General Options' in exports(5)
+NFS4SERVER_EXPORT_OPTS=("rw" "no_root_squash" "async" "no_subtree_check")
+# List of trusted systems that can mount the recovery system, see 'Machine Name Formats' in exports(5)
+# e.g. NFS4SERVER_TRUSTED_CLIENTS=( myserver 172.16.76.0/24 )
```

Comment:
I am not a NFS expert so I may confuse something but
```
NFS4SERVER_TRUSTED_CLIENTS=( myserver ...)
```
looks contradicting i.e. a trusted client called 'myserver'?

Reply:
With NFS, a "client" connects to a "server" who authorises the "client" during connection. Therefore the term "allowed clients" or "permitted clients" or even just "clients" would probably be most correct.